src/Page.js

```diff
@@ -849,6 +852,22 @@ Page.prototype.postRender = function (content) {
   return postRenderedContent;
 };
 
+/**
+ * Collect page content inserted by plugins
+ */
+Page.prototype.collectPluginsContent = function () {
+  let links = [];
+  let scripts = [];
+
+  Object.entries(this.plugins).forEach(([pluginName, plugin]) => {
+    if (plugin.addLinks) {
+      links = links.concat(plugin.addLinks(this.pluginsContext[pluginName], this.frontMatter));
+      scripts = scripts.concat(plugin.addScripts(this.pluginsContext[pluginName], this.frontMatter));
+    }
+  });
+  this.pluginsContent = { links, scripts };
```

Comment:
Put in `this.asset.pluginScripts` since there's `this.asset.externalScripts`.
Similar for links.